plugins/MeteorShowers/src/gui/MSConfigDialog.hpp

```diff
@@ -71,8 +74,9 @@ private slots:
 private:
 	MeteorShowersMgr* m_mgr;
 	Ui_MSConfigDialog* m_ui;
-
+	QTimer* updateTimer;
```

Comment:
There is a m_updateTimer in the MeteorShowerMgr.hpp. Are they fulfilling different purposes? (I have not built/tested the code yet)

Reply:
Thanks for pointing it out. In my understanding, they're unrelated and no need to declare here.